appveyor.yml

```diff
@@ -20,7 +19,7 @@ before_build:
   - cmd: '"C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\sn.exe" -k winsw_key.snk'
 
 dotnet_csproj:
-  patch: true
+  # patch: false
```

Comment:
It is not going to fix the issue, because we still need to patch the release version. I will try to find alternate way

Reply:
TL;DR: UI does not override this setting AFAICT

Reply:
> UI does not override this setting AFAICT

Then move the entire `dotnet_csproj:` sections into release configuration?